cmd/geth/chaincmd.go

```diff
@@ -328,6 +343,38 @@ func parseIps(ipStr string, size int) ([]string, error) {
 	return ips, nil
 }
 
+func parsePorts(portStr string, defaultPort int, size int) ([]int, error) {
+	var ports []int
+	if len(portStr) != 0 && strings.Contains(portStr, ",") {
```

Comment:
line 360 is `len(portStr) != 0`
so here IMO
     len(portStr) != 0 && strings.Contains(portStr, ",") --> strings.Contains(portStr, ",")